assets/tests/QuestionSchemaValidationService.js

```diff
@@ -60,6 +60,23 @@ tie.factory('QuestionSchemaValidationService', [
           angular.isArray(question.getTasks()) &&
           question.getTasks().length > 0);
       },
+      verifyAtLeastOneBuggyOutputTestExists: function(question) {
```

Comment:
I'm not 100% sure that this is an absolute requirement, but happy for you to make the call.